ee/clickhouse/views/test/__snapshots__/test_clickhouse_stickiness.ambr

```diff
@@ -125,6 +127,7 @@
        AND timestamp <= '2020-02-15 23:59:59'
        AND event = 'watched movie'
        AND NOT has([''], $group_0)
+       AND NOT has([''], $group_0)
```

Comment:
updated get_target_entity and it revealed an oddity with `test_aggregating_by_group`. The entity property filters actually have a repeated one. Previous get_target_entity wasn't returning the one with duplicated filters but now it is

Reply:
What's the action step here?

Reply:
Bit of clean up that needs to happen. It exists in more tests than just the one above so I noted the issue https://github.com/PostHog/posthog/issues/7745 and will do a clean up separately